tests/php/Integration/Run/Command/Run/RunCommandTest.php

```diff
@@ -25,12 +25,12 @@ public static function setUpBeforeClass(): void
      *
      * @preserveGlobalState disabled
      */
-    public function test_run_by_namespace(): void
+    public function test_file_not_found(): void
     {
-        $this->expectOutputRegex('~hello world~');
+        $this->expectOutputString('File "non-existing-file.phel" not found' . PHP_EOL);
 
         $this->createRunCommand()->run(
-            $this->stubInput('test\\test-script'),
+            $this->stubInput('non-existing-file.phel'),
```

Comment:
Why not keeping both tests? Did you override it intentionally? 🤔

Reply:
@Chemaclass the old one doesn't make sense anymore, so I removed it and created the other one.

I mentioned the new logic and also this test in the PR description over the `investigation` part.